templates/game.eex

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="utf-8">
+    <title>Elixir TicTacToe</title>
+    <style>
+
+body {
+  background-color: black;
+  color: white;
+}
+
+h1 {
+  text-align: center;
+}
+
+#board {
+  background-color: #4d4d4d;
```

Comment:
Is there a reason why you didn't seperate the CSS out into a stylesheet? I think it would be better not to have html and css mixed together into one file...